projects/helper/coreAssets.json

```diff
@@ -1844,7 +1844,13 @@
     "STETH": "0x85008aE6198BC91aC0735CB5497CF125ddAAc528",
     "STONE": "0x96147a9ae9a42d7da551fd2322ca15b71032f342",
     "RETH": "0xb5686c4f60904ec2bda6277d6fe1f7caa8d1b41a",
-    "TBTC": "0xbba2ef945d523c4e2608c9e1214c2cc64d4fc2e2"
+    "TBTC": "0xbba2ef945d523c4e2608c9e1214c2cc64d4fc2e2",
+    "SolvBTC": "0x541fd749419ca806a8bc7da8ac23d346f2df8b77",
```

Comment:
so, this file was never meant to be an address book, but feel like it is a losing battle atm, and I am throwing in the towel.